contentcuration/contentcuration/migrations/0103_auto_20190905_0408.py

```diff
@@ -16,11 +16,11 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='fileformat',
             name='extension',
-            field=models.CharField(choices=[(b'mp4', b'MP4 Video'), (b'vtt', b'VTT Subtitle'), (b'mp3', b'MP3 Audio'), (b'pdf', b'PDF Document'), (b'jpg', b'JPG Image'), (b'jpeg', b'JPEG Image'), (b'png', b'PNG Image'), (b'gif', b'GIF Image'), (b'json', b'JSON'), (b'svg', b'SVG Image'), (b'perseus', b'Perseus Exercise'), (b'graphie', b'Graphie Exercise'), (b'zip', b'HTML5 Zip'), (b'h5p', b'H5P'), (b'epub', b'ePub Document')], max_length=40, primary_key=True, serialize=False),
```

Comment:
What were the actual fixups made in these migrations?

Reply:
Just turning bytestring into a regular string.

Reply:
@aronasorman If you were asking what the actual change in the migration is, it is getting generated because the le-utils constants (`fileformat` in this case) changed, and those constants are used for validation in the db field. So every time the constants change in le-utils, we need to generate a new migration to update the db validation.